internal/executor/reporter/testfixtures.go

```diff
@@ -0,0 +1,25 @@
+package reporter
```

Comment:
As above, you probably want to name this testfixtures_test.go or move into a dedicated fixtures package.